.circleci/config.yml

```diff
@@ -3,6 +3,16 @@ version: 2.1
 orbs:
   hokusai: artsy/hokusai@0.7.5
   horizon: artsy/release@0.0.1
+  artsy-remote-docker: artsy/remote-docker@dev:25a57d9dad11f4ada9c4d9e42eb691f0
```

Comment:
Should be be set to the newly deploy orb version instead of a hash?

Reply:
Absolutely